### PR TITLE
Gem smitamishra

### PR DESCRIPTION
**Pull request by gem-smitamishra**

### CODE REVIEW

**Comment by gem-smitamishra** [comment]
@gemAkshay Please review this

File: src/main/java/com/qa/mis/stepdefinition/DashboardLeaveBalanceSteps.java [hunk 1]
```diff
@@ -14,22 +15,26 @@
...
-    public void verifySection(){
+    public void verifySection() {
         DriverAction.waitSec(5);
         DriverAction.scrollIntoView(DashboardLeaveBalanceLocator.leaveBalanceSection);
-        DriverManager.getWebDriver().manage().timeouts().implicitlyWait(Duration.ofSeconds(10));
-        WebDriverWait wait = new WebDriverWait(DriverManager.getWebDriver(), 20);
-        wait.until(ExpectedConditions.presenceOfElementLocated(DashboardLeaveBalanceLocator.leaveheading));
-        if (DriverAction.isExist(DashboardLeaveBalanceLocator.leaveheading)) {
-            GemTestReporter.addTestStep("Verify Section", "Leave Section is present on the current page", STATUS.PASS, DriverAction.takeSnapShot());
+        presenceOfElement();
```
Comment (gemAkshay) [inline src/main/java/com/qa/mis/stepdefinition/DashboardLeaveBalanceSteps.java:27]: Send the Location of element to be checked for presence

File: src/main/java/com/qa/mis/stepdefinition/DashboardLeaveBalanceSteps.java [hunk 2]
```diff
@@ -43,30 +48,28 @@ public void clickOnNumberOfLeaves() {
...
-        WebDriverWait wait = new WebDriverWait(DriverManager.getWebDriver(), 20);
-        wait.until(ExpectedConditions.presenceOfElementLocated(DashboardLeaveBalanceLocator.leavesRecords));
+        presenceOfElement();
         if (DriverAction.isExist(DashboardLeaveBalanceLocator.leavesRecords)) {
-            String records= DriverAction.getElementText(DashboardLeaveBalanceLocator.leavesRecords);
-          //  leaveCount = "0";
-            if(records.contains(leaveCount)){
+            String records = DriverAction.getElementText(DashboardLeaveBalanceLocator.leavesRecords);
+
+            if (records.contains(leaveCount)) {
```
Comment (gemAkshay) [inline src/main/java/com/qa/mis/stepdefinition/DashboardLeaveBalanceSteps.java:65]: How can this has been passed? leaveCount wasn't initialized

Comment (gem-smitamishra) [reply src/main/java/com/qa/mis/stepdefinition/DashboardLeaveBalanceSteps.java:65]: leave count is a global variable for entire class and value will be assign at line 42

**Review by gemAkshay — APPROVED** [review]
@gemini-akashGarg No Review required as asked by Smita

**Review by gemini-akashGarg — APPROVED** [review]

**Pull request merged**